tests/cross_fw/shared/command.py

```diff
@@ -17,7 +17,9 @@
 from pathlib import Path
 from typing import Any, Dict, List
 
-from nncf.common.utils.os import is_windows
+
+def is_windows() -> bool:
+    return "win32" in sys.platform
```

Comment:
Why it was replaced? 
It looks like an unnecessary change since it is a copy of `nncf.common.utils.os.is_windows`.

Reply:
To remove dependencies (for only utils function) from NNCf, and avoid install NNCF for cross_fw/examples tests.